types/options.go

```diff
@@ -278,6 +278,9 @@ func ReloadConfigurationFile(configFile string, storeOptions *StoreOptions) {
 		fmt.Printf("Failed to parse %s %v\n", configFile, err.Error())
 		return
 	}
+
+	// Clear storeOptions of previos settings
+	storeOptions = StoreOptions{}
```

Comment:
should this be `storeOptions = &StoreOptions{}` ?

Reply:
Yup